samtranslator/internal/schema_source/aws_serverless_function.py

```diff
@@ -280,6 +281,7 @@ class ApiEventProperties(BaseModel):
     RequestParameters: Optional[RequestModelProperty] = apieventproperties("RequestParameters")
     RestApiId: Optional[Union[str, Ref]] = apieventproperties("RestApiId")
     TimeoutInMillis: Optional[PassThroughProp]  # TODO: add doc
+    TestAuthOverride: Optional[PassThroughProp]  # TODO: add doc
```

Comment:
Why's it here aswell? Isn't it just under auth?

Reply:
Good catch, will remove this.